torch/csrc/autograd/anomaly_mode.cpp

```diff
@@ -0,0 +1,49 @@
+#include "torch/csrc/python_headers.h"
+#include "torch/csrc/autograd/anomaly_mode.h"
+#include "torch/csrc/utils/auto_gil.h"
+#include "torch/csrc/utils/python_strings.h"
+#include "torch/csrc/utils/object_ptr.h"
+#include "torch/csrc/Exceptions.h"
+
+#include <stdexcept>
+#include <iostream>
+
+namespace torch { namespace autograd {
+
+bool AnomalyMode::_enabled = 0;
+PyObject* AnomalyMode::stacktrace_key = THPUtils_packString("stacktrace_");
```

Comment:
This feels dirty. I know we leak Python objects in other places, but I'd rather avoid it.

Just use `PyDict_SetItemString` with a `string literal` or `const char*` instead of `PyDict_SetItem`

Reply:
Ho I did not know that existed, will do the change.